### ISSUE TITLE
unable to display docs for (sub)packages on gitlab 

### ISSUE THREAD

**Issue by sbinet** [golang/gddo#374]
somewhat related to golang/gddo#356 I am experiencing the following:

```sh
$> curl 'https://gitlab.in2p3.fr/sbinet/hbook?go-get=1'
<!DOCTYPE html>
<html>
<head>
<meta content='gitlab.in2p3.fr/sbinet/hbook git https://gitlab.in2p3.fr/sbinet/hbook.git' name='go-import'>
</head>
</html>

$> curl 'https://gitlab.in2p3.fr/sbinet/hbook/subpkg?go-get=1'
<html><body>You are being <a href="https://gitlab.in2p3.fr/users/sign_in">redirected</a>.</body></html>
```

ie: it works for `gitlab.in2p3.fr/sbinet/hbook` but not for `gitlab.in2p3.fr/sbinet/hbook/subpkg`

https://godoc.org/gitlab.in2p3.fr/sbinet/hbook
https://godoc.org/gitlab.in2p3.fr/sbinet/hbook/subpkg

interestingly, "this time", it doesn't work either with gotools.org:
https://gotools.org/gitlab.in2p3.fr/sbinet/hbook/subpkg

-s

**Comment by sbinet** [comment]
@ebusato, it seems it doesn't come from your package per-se :)

**Comment by shurcooL** [comment]
> ```bash
> $> curl 'https://gitlab.in2p3.fr/sbinet/hbook/subpkg?go-get=1'
> <html><body>You are being <a href="https://gitlab.in2p3.fr/users/sign_in">redirected</a>.</body></html>
```

That's where the problem is. It has nothing to do with [godoc.org](https://godoc.org) or (gotools.org)(https://godoc.org), they are correctly failing to display the Go package since the `?go-get=1` response is invalid.

**Comment by sbinet** [comment]
fair enough.
do you know what I should tell my sysadmin? or even better, to where I should point her?

**Comment by shurcooL** [comment]
I think the issue is in GitLab. You should update to the latest version (if not already), in case this issue is already fixed there, and if not, create an issue in their issue tracker.

**Comment by fiorix** [comment]
Fix is in Gitlab 8.6 as per https://github.com/gitlabhq/gitlabhq/issues/9508.
